docs/diagnostics/PX1035.md

```diff
@@ -0,0 +1,39 @@
+# PX1035
+This document describes the PX1035 diagnostic.
+
+## Summary
+
+| Code   | Short Description                                 | Type  | Code Fix    | 
+| ------ | ------------------------------------------------- | ----- | ----------- | 
+| PX1035 | The DAC has multiple primary key declarations | Warning (ISV Level 3: Informational) | Unavailable | 
+
+## Diagnostic Description
+A DAC cannot have multiple primary key declarations. The warning is shown only for DACs that have either PXCacheName or PXPrimaryGraph attribute.
+
+To fix the issue, you should remove all but one primery key declarations.
```

Comment:
The code fix should be described. It removes all primary key declarations except the selected one